persistence/remote/src/test/resources/remote-cl-uri-config.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<infinispan
+      xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+      xsi:schemaLocation="urn:infinispan:config:${infinispan.core.schema.version} https://infinispan.org/schemas/infinispan-config-${infinispan.core.schema.version}.xsd
+                          urn:infinispan:config:store:remote:${infinispan.core.schema.version} https://infinispan.org/schemas/infinispan-cachestore-remote-config-${infinispan.core.schema.version}.xsd"
+      xmlns="urn:infinispan:config:${infinispan.core.schema.version}"
+      xmlns:remote="urn:infinispan:config:store:remote:${infinispan.core.schema.version}" >
+
+   <!-- Default cache named to preserve old default cache name -->
+   <cache-container default-cache="RemoteStoreConfigUriTest">
+      <local-cache name="RemoteStoreConfigUriTest">
+         <persistence passivation="false">
+            <remote-store xmlns="urn:infinispan:config:store:remote:${infinispan.core.schema.version}"
+                          cache="RemoteStoreConfigUriTest" fetch-state="false" preload="false" purge="false" shared="true" read-only="false"
+                          hotrod-wrapping="false" raw-values="false" socket-timeout="60000"
+                          tcp-no-delay="true" balancing-strategy="org.infinispan.client.hotrod.impl.transport.tcp.RoundRobinBalancingStrategy"
+                          transport-factory="org.infinispan.client.hotrod.impl.transport.netty.ChannelFactory"
+                          key-size-estimate="62" value-size-estimate="512" force-return-values="false" segmented="false" uri="hotrod://127.0.0.1:19711">
```

Comment:
We should probably configure multiple servers in the URI for completeness.

Reply:
I am testing with more options, will update it as final uri.